mbzirc_custom/mbzirc_naive_3d_scanning_radar/models/sensors/mbzirc_naive_3d_scanning_radar/model.sdf

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<sdf version="1.9">
+  <model name="mbzirc_naive_3d_scanning_radar">
+    <link name="base_link">
+      <inertial>
+        <mass>0.005</mass>
+        <inertia>
+          <ixx>8.33e-06</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>8.33e-06</iyy>
+          <iyz>0</iyz>
+          <izz>8.33e-06</izz>
+        </inertia>
+      </inertial>
+      <sensor name='gpu_lidar' type='gpu_lidar'>
+        <update_rate>1</update_rate>
+        <lidar>
+          <scan>
+            <horizontal>
+              <samples>250</samples>
+              <resolution>1</resolution>
+              <min_angle>-3.14159</min_angle>
+              <max_angle>3.14159</max_angle>
+            </horizontal>
+            <vertical>
+              <samples>32</samples>
+              <resolution>1</resolution>
+              <min_angle>-0.698132</min_angle>
+              <max_angle>0.0872665</max_angle>
+            </vertical>
+          </scan>
+          <range>
+            <min>1</min>
+            <max>1500</max>
```

Comment:
A variety of of radar models were proposed by the teams and one common aspect is that they plan to use models with a longer range in the physical competition that is sufficient to cover the competition area (10km^2). So one proposal is to extend the max range to cover the competition area and adjust the resolution and noise accordingly. The downside is that it no longer matches the spec for the particular model chosen. I can make this change

Reply:
ok extended range in [bdb622b](https://github.com/osrf/mbzirc/pull/161/commits/bdb622be63d3179a68d71baa0d22facfe3243c44)